src/StudentOrganizer.Infrastructure/Commands/Groups/EditGroupName.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Text.Json.Serialization;
+
+namespace StudentOrganizer.Infrastructure.Commands.Groups
```

Comment:
Nie dodałeś tej klasy, what the huskwiernia?

Reply:
lol, powinno wszystko się dodać